index.js

```diff
@@ -22,10 +22,19 @@ var waitForSingleFileChange = (file, interval) =>
     checkFile();
   });
 
-module.exports = (files, interval) => {
-  if (typeof files === "string") {
-    files = [files];
+module.exports = (args) => {
```

Comment:
I would rather not change the API here, but have some dedicated logic for that within the binary [wait-for-change.js](https://github.com/svi3c/wait-for-change/blob/master/wait-for-change.js).
We are using this API not only from the CLI, but also programmatically and for those cases the original API makes much more sense imo.